benchmarks/asv.conf.json

```diff
@@ -35,7 +35,7 @@
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
-    "pythons": ["2.7"],
+    "pythons": [ "3.6" ],
```

Comment:
Hmm... A change like this probably belongs in a separate PR. @pv Does this impact you?